tests/timezones/tests.py

```diff
@@ -1231,3 +1232,20 @@ def test_change_readonly_in_other_timezone(self):
         with timezone.override(ICT):
             response = self.client.get(reverse('admin_tz:timezones_timestamp_change', args=(t.pk,)))
         self.assertContains(response, t.created.astimezone(ICT).isoformat())
+
+
+@override_settings(TIME_ZONE='America/New_York', USE_TZ=True)
+class FExpression(TestCase):
+    def setUp(self):
+        self.timestamp = ArbitraryTimestamp()
+
+    def test_F_expression(self):
+        now = timezone.now()
+        self.timestamp.updated = now - timedelta(hours=2)
```

Comment:
Try to minimize the test that demonstrates the regression. I think this part isn't important -- assigning a value when creating the model should work just as well.